benchmarks/src/main/scala/zio/stm/TReentrantLockBenchmark.scala

```diff
@@ -12,8 +12,8 @@ import zio._
 @State(Scope.Group)
 @BenchmarkMode(Array(Mode.Throughput))
 @OutputTimeUnit(TimeUnit.SECONDS)
-@Measurement(iterations = 15, timeUnit = TimeUnit.SECONDS, time = 10)
-@Warmup(iterations = 15, timeUnit = TimeUnit.SECONDS, time = 10)
+@Measurement(iterations = 5, timeUnit = TimeUnit.SECONDS, time = 5)
```

Comment:
I wouldn't decrease these.

Reply:
just as a quick and dirty check speed check - I'll bump them back up at the end of the other pr